sql/core/src/test/resources/sql-tests/results/show-create-table.sql.out

```diff
@@ -78,7 +78,7 @@ CREATE TABLE spark_catalog.default.tbl (
   b STRING,
   c INT)
 USING parquet
-LOCATION 'file:///path/to/table'
+LOCATION 'file:/path/to/table'
```

Comment:
I've investigated this locally. The new result is actually consistent with the production behavior (with Hive Metastore). What happens is:
- CREATE TABLE command qualifies the table location, and keeps it as `URI` in `CatalogTable#storage#locationUri`
- When saving the table into HMS, we have to turn `URI` into string, by using `new Path(uri).toString`. The Hadoop Path string will omit the `authority` component if it's not present. So `file:///path` becomes `file:/path`.
- When reading the table back from HMS, we turn string back to `URI`, but the `authority` component won't be filled.

However, with `InMemoryCatalog`, we keep `CatalogTable` directly and there is no `URI` <-> string round trip. So the `authority` component is still there.

This actually doesn't matter, as empty `authority` is the same as no `authority`, in `URI` string.

Reply:
Thank you for the details.